observation_portal/requestgroups/viewsets.py

```diff
@@ -77,7 +77,9 @@ def schedulable_requests(self, request):
         current_semester = Semester.current_semesters().first()
         start = parse(request.query_params.get('start', str(current_semester.start))).replace(tzinfo=timezone.utc)
         end = parse(request.query_params.get('end', str(current_semester.end))).replace(tzinfo=timezone.utc)
-        telescope_class = request.query_params.get('telescope_class')
+        telescope_classes = request.query_params.get('telescope_classes')
```

Comment:
To be more in line how our other endpoints that allow multiple values for certain query params work, what do you think about letting a user set the `telescope_class` param one or more times (so, for example, `?telescope_class=0m4&telescope_class=1m0`), and then from here get the list of telescope classes using the [QueryDict's getlist method](https://docs.djangoproject.com/en/3.1/ref/request-response/#querydict-objects)?

Reply:
didn't know about the getlist method on querydicts but that does sound better. I'll see about switching it over

Reply:
okay, changed the keyword to `telescope_class` using getlist for multiples